tests/databases/api_tests.py

```diff
@@ -95,7 +95,7 @@ def test_get_items(self):
             "function_names",
             "id",
         ]
-        self.assertEqual(response["count"], 2)
```

Comment:
`count = 2` seems to assume some tests run before this test. If you just initialized the database and run this test alone, this test will fail.